.dockerignore

```diff
@@ -1,22 +1,13 @@
-alice
-bob
-.dockerignore
-node_modules
-scripts
-test
-zero
-*.db
-charts
-helmfile.d
-build
-.circleci
-.editorconfig
-.git
-.github
-.gitignore
-.idea
-combined.log
-docker-compose.yml
-node_modules/
-
-*/**/node_modules/
\ No newline at end of file
+*
+!apps
+!candidates
+!docs
+!packages
+!scripts
+!package.json
+!tsconfig.json
+!turbo.json
+!.yarnrc.yml
+!.yarn
+!yarn.lock
+.yarn/cache
```

Comment:
.yarn/cache ?

Reply:
We need `.yarn/releases` and `.yarn/plugins`, but we don't need `.yarn/cache`. 
However, now I think that it's more appropriate to add `!.yarn/releases` and `!.yarn/plugins` directly, and omit ignoring `.yarn/cache`